LibreHardwareMonitorLib/LibreHardwareMonitorLib.csproj

```diff
@@ -28,10 +29,16 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
   <ItemGroup>
```

Comment:
I'll do a more thorough review later, but I wonder if we use a submodule here instead of manually copying these every time. Or create a NuGet package that we can pull in.

Reply:
I think RAMSPDToolkit relies on API/ABI stability of the SMBus modules, while LHM depends on the rest. So it is possible that in some future scenarios you’d want to mix multiple “releases”.

I should probably do semver with each release being “breaking” (ie major version bump) and document which modules actually changed API/ABI. The idea is that everyone just ships whatever they’re working with so a stable API is less important.

Reply:
> I think RAMSPDToolkit relies on API/ABI stability of the SMBus modules, while LHM depends on the rest. So it is possible that in some future scenarios you’d want to mix multiple “releases”.
> 
> I should probably do semver with each release being “breaking” (ie major version bump) and document which modules actually changed API/ABI. The idea is that everyone just ships whatever they’re working with so a stable API is less important.

Ah, yes, that's a downside indeed, although I wonder how often this will be needed. I thought it'd be nice to be able to point to a verified commit, rather than someone's upload, but we can also manually update instead.